Userland/Libraries/LibVideo/PlaybackManager.cpp

```diff
@@ -52,9 +52,19 @@ DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> PlaybackManager::create(NonnullOw
 
     dbgln_if(PLAYBACK_MANAGER_DEBUG, "Selecting video track number {}", track.identifier());
 
-    auto decoder = DECODER_TRY_ALLOC(try_make<VP9::Decoder>());
+    OwnPtr<VideoDecoder> decoder;
+    switch (track.codec_id()) {
+    case CodecID::VP9:
+        decoder = DECODER_TRY_ALLOC(try_make<VP9::Decoder>());
+        break;
+
+    default:
+        dbgln("Unsupported codec: {}", (int)track.codec_id());
```

Comment:
```suggestion
        dbgln("Unsupported codec: {}", to_underlying(track.codec_id()));
```

But a formatter would be even better.